rules/Removing/Rector/FuncCall/RemoveFuncCallRector.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Removing\Rector\FuncCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Stmt\Expression;
+use Rector\Core\Contract\Rector\ConfigurableRectorInterface;
+use Rector\Core\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\ConfiguredCodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+use Webmozart\Assert\Assert;
+
+/**
+ * @see \Rector\Tests\Removing\Rector\FuncCall\RemoveFuncCallRector\RemoveFuncCallRectorTest
+ */
+final class RemoveFuncCallRector extends AbstractRector implements ConfigurableRectorInterface
+{
+    /**
+     * @var string[]
+     */
+    private array $removedFunctions = [];
+
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition('Remove function', [
+            new ConfiguredCodeSample(
+                <<<'CODE_SAMPLE'
+$x = 'something';
+var_dump($x);
+CODE_SAMPLE
+                ,
+                <<<'CODE_SAMPLE'
+$x = 'something';
+CODE_SAMPLE
+                ,
+                ['var_dump']
+            ),
+        ]);
+    }
+
+    /**
+     * @return array<class-string<Node>>
+     */
+    public function getNodeTypes(): array
+    {
+        return [FuncCall::class, Expression::class];
```

Comment:
FuncCall need to be removed:

```suggestion
        return [Expression::class];
```